docs/how-to/enforce-permissions/bulk-check.mdx

```diff
@@ -162,6 +162,18 @@ const permittedResources = resources.filter(
 ```
 
 ---
+### Set Timeout
+For setting the timeout for the bulk check, you can set the 'PDP_OPA_CLIENT_QUERY_TIMEOUT' environment variable for the PDP.
```

Comment:
Bulk check is more performance intensive than regular check and also takes more time which can lead to OPA timeout.
To address this issue, you can increase the query timeout for the PDP and add more CPU to your VM.